app/src/main/res/layout/fragment_car_drivingstats.xml

```diff
@@ -1,449 +1,705 @@
-                    android:layout_height="wrap_content"
-                    android:layout_weight="1"
-                    android:text="@string/lbl_carinfo_lastchecked"/>
-                <TextView
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_weight="1"
-                    android:text="@{viewModel.lastUpdate}"/>
-            </LinearLayout>
-
-            <!-- Position -->
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical"
-                android:visibility="@{viewModel.positionGeoUri != null}">
-                <LinearLayout
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:orientation="horizontal">
-                    <TextView
-                        android:layout_width="0dp"
-                        android:layout_height="wrap_content"
-                        android:layout_weight="1"
-                        android:text="@string/lbl_carinfo_lastPosition"/>
-
-                    <androidx.constraintlayout.widget.ConstraintLayout
-                        android:id="@+id/linearLayout"
-                        android:layout_width="0dp"
-                        android:layout_height="wrap_content"
-                        android:layout_weight="1"
-                        android:clickable="true"
-                        android:focusable="true"
-                        android:onClick="@{() -> controller.viewUri(viewModel.positionGeoUri)}">
-
-                        <TextView
-                            android:id="@+id/txtPositionName"
-                            android:layout_width="0dp"
-                            android:layout_height="wrap_content"
-                            android:text="@{viewModel.positionName}"
-                            app:layout_constraintEnd_toStartOf="@+id/icMapPin"
-                            app:layout_constraintStart_toStartOf="parent"
-                            app:layout_constraintTop_toTopOf="parent"
-                            />
-
-                        <TextView
-                            android:id="@+id/txtGeoName"
-                            android:layout_width="0dp"
-                            android:layout_height="wrap_content"
-                            android:text="@{viewModel.positionGeoName}"
-                            android:textSize="12sp"
-                            android:visibility="@{viewModel.showAdvancedSettings}"
-                            app:layout_constraintEnd_toStartOf="@+id/icMapPin"
-                            app:layout_constraintStart_toStartOf="parent"
-                            app:layout_constraintTop_toBottomOf="@+id/txtPositionName"/>
-
-                        <ImageButton
```

Comment:
I changed this to an ImageButton after you started your branch, I apologize!

Reply:
ok, updated my file.